website/blog/2021-09-06-tungsten.md

```diff
@@ -0,0 +1,358 @@
+---
+author: tgodzik
+title: Metals v0.10.6 - Tungsten
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We're happy to announce the release of Metals v0.10.6, which brings about new
+Scala 3 versions and JDK 17 support in addition to some new interesting
+features.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">173</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">74</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">16</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">25</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">4</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/41?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Adjust indentation on paste.
+- New code actions.
+- [Scala 3] support for Scala 3.0.2 and 3.1.0-RC1.
+- Allow organize imports on unsaved code.
+- JDK 17 support.
+
+## Adjust Indentation on paste
+
+Starting with Scala 3 developers can use optional braces, which we mentioned in
+some of the previous release notes. One of the main problems with it is making
+sure that users are able to properly paste code into existing blocks. Previously
+we could have just pasted into a braced block and adjust indentation later, but
+now it really matters that indentation is changed properly in order to make to
+code compile correctly.
+
+To help with that [@Giggiux](https://github.com/Giggiux) adjusted the existing
+range formatting to account for indentation in the pasted blocks. Starting from
```

Comment:
I believe this only works with editors that support range formatting correct? We may want to mention that.

Reply:
Good idea, added a mention at the end of the section.